calculator-project/src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react'
+
+class Display extends Component {
```

Comment:
I'd recommend using a functional component since Display doesn't need to manage its own state or use lifecycle methods